sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,736 @@
+            raise ValueError(
+                "A signing certificate and key must be provided to add_policy_management_certificate."
+            )
+
+        # Verify that the provided certificate is a valid PEM encoded X.509 certificate
+        certificate_to_add = load_pem_x509_certificate(
+            certificate_to_add.encode("ascii")
+        )
+
+        jwk = JSONWebKey(
+            kty="RSA",
+            x5_c=[
+                base64.b64encode(
+                    certificate_to_add.public_bytes(serialization.Encoding.DER)
+                ).decode("ascii")
+            ],
+        )
+        add_body = AttestationCertificateManagementBody(policy_certificate=jwk)
+        cert_add_token = AttestationToken(
+            body=add_body,
+            signing_key=signing_key,
+            signing_certificate=signing_certificate,
+            body_type=AttestationCertificateManagementBody,
+        )
+
+        # Merge our existing config options with the options for this API call.
+        # Note that this must be done before calling into the implementation
+        # layer because the implementation layer doesn't like keyword args that
+        # it doesn't expect :(.
+        options = merge_validation_args(
+            self._config._kwargs, kwargs  # pylint: disable = protected-access
+        )
+
+        cert_response = self._client.policy_certificates.add(
+            cert_add_token.to_jwt_string(), **kwargs
+        )
+        token = AttestationToken(
+            token=cert_response.token,
+            body_type=GeneratedPolicyCertificatesModificationResult,
+        )
+
+        if options.get("validate_token", True):
+            token._validate_token(  # pylint: disable = protected-access
+                self._get_signers(**kwargs), **options
+            )  # pylint: disable = protected-access
+        return (
+            AttestationPolicyCertificateResult._from_generated(  # pylint: disable = protected-access
+                token._get_body()  # pylint: disable = protected-access
+            ),
+            token,
+        )
 
     @distributed_trace
-    def remove_policy_management_certificate(self, certificate_to_add, signing_key, **kwargs):
-        #type:(bytes, AttestationSigningKey, **Any) -> AttestationResponse[PolicyCertificatesModificationResult]
-        """ Removes a new policy management certificate to the set of policy management certificates for the instance.
+    def remove_policy_management_certificate(self, *args, **kwargs):
+        # type: (*str, **Any) -> Tuple[AttestationPolicyCertificateResult, AttestationToken]
+        # pylint: disable = line-too-long
+        """Removes a policy management certificate from the set of policy management certificates for the instance.
 
-        :param bytes certificate_to_add: DER encoded X.509 certificate to add to 
+        :param str certificate_to_remove: PEM encoded X.509 certificate to remove from
```

Comment:
Again, probably good to emphasize that this is required
```suggestion
        :param str certificate_to_remove: Required. PEM encoded X.509 certificate to remove from
```